client/src/app/menu/menu.component.html

```diff
@@ -100,32 +131,56 @@
           <ng-container i18n>Local</ng-container>
         </a>
       </div>
+    </div>
 
+    <div class="footer">
       <div class="panel-block">
-        <div class="block-title" i18n>More</div>
-
         <a *ngIf="userHasAdminAccess" [routerLink]="getFirstAdminRouteAvailable()" routerLinkActive="active">
-          <my-global-icon iconName="administration"></my-global-icon>
+          <my-global-icon iconName="cog"></my-global-icon>
           <ng-container i18n>Administration</ng-container>
         </a>
-
-        <a routerLink="/about" routerLinkActive="active">
-          <my-global-icon iconName="about"></my-global-icon>
+        <a *ngIf="!isLoggedIn" (click)="openQuickSettings()">
+          <my-global-icon iconName="cog"></my-global-icon>
+          <ng-container i18n>Settings</ng-container>
+        </a>
+        <a routerLink="/about/instance">
+          <my-global-icon iconName="help"></my-global-icon>
           <ng-container i18n>About</ng-container>
         </a>
       </div>
-    </div>
-
-    <div class="footer d-flex justify-content-between">
-      <span class="language">
-        <span tabindex="0" role="button" (keyup.enter)="openLanguageChooser()" (click)="openLanguageChooser()" i18n-title title="Change the language" class="icon icon-language"></span>
-      </span>
 
-      <span class="shortcuts">
-        <span tabindex="0" role="button" (keyup.enter)="openHotkeysCheatSheet()" (click)="openHotkeysCheatSheet()" i18n-title title="Show keyboard shortcuts" class="icon icon-shortcuts"></span>
-      </span>
+      <div class="d-flex flex-column" style="row-gap: 1em;">
+        <div id="links-primary">
+          <a i18n routerLink="/about/instance">Contact</a>
+          <a i18n routerLink="/about/instance">Terms of Service</a>
+          <a i18n routerLink="/about/instance" fragment="statistics">Stats</a>
+          <a (click)="openLanguageChooser()" class="c-hand">
+            <span i18n>Interface: {{ language }}</span>
+          </a>
+        </div>
+        <div id="links-secondary">
+          <a i18n href="https://joinpeertube.org/#you-are-a-video-maker" i18n-title title="Creator guide" target="_blank" rel="noopener noreferrer">Creators</a>
+          <a i18n href="https://docs.joinpeertube.org/#/contribute-getting-started" i18n-title title="PeerTube license" target="_blank" rel="noopener noreferrer">Contributors</a>
```

Comment:
It should be "Contribute" no? If you want to keep "Contributors" it should link the credits page

Reply:
No, I think "Contributors" is fine, since we use names and not verbs in the links so far. Also, that would mean changing "Creators" to "Create", which is less than ideal.

Reply:
Okay I understand the logic. Let's try like that